### DISCUSSION TITLE
OpenRouter and Perplexity support

### DISCUSSION THREAD

**Discussion by zheroz00** [ILikeAI/AlwaysReddy#30] [Ideas]
Is it possible to add OpenRouter and Perplexity access? Would it be possible to use OpenRouter for the completions and transcriptions? I Thanks

**Comment by ILikeAI** [comment]
Omg I hadnt been getting notified of my discussions, sorry I didnt see this till now! I actually just added perplexity support like 2 seconds ago, have fun :)

**Comment by ILikeAI** [comment]
Just added open router support too but couldn't really test it because they wouldnt let me buy credits for some reason
